src/components/DevconGrantsForm.js

```diff
@@ -0,0 +1,775 @@
+import React, { useState } from "react"
+import { navigate } from "gatsby"
+import { useIntl } from "gatsby-plugin-intl"
+import { FormattedMessage } from "gatsby-plugin-intl"
+import styled from "styled-components"
+import { useToasts } from "react-toast-notifications"
+import Select from "react-select"
+
+import { validateEmail } from "../utils/validate-email"
+import {
+  Form,
+  Label,
+  Input,
+  Checkbox,
+  CheckboxInput,
+  TextArea,
+  Button,
+  Required,
+} from "./SharedStyledComponents"
+import { colorRed } from "../utils/styles"
+
+const StyledForm = styled(Form)`
+  margin-top: 2rem;
+`
+
+const StyledSelect = styled(Select)`
+  margin-top: 0.5rem;
+`
+
+const StyledLabel = styled(Label)`
+  margin-bottom: 2.5rem;
+`
+
+const ErrorMessage = styled.small`
+  color: ${colorRed};
+`
+
+const ErrorDiv = styled.div`
+  margin-top: 0.5rem;
+  height: 0rem;
+  width: 100%;
+`
+
+const StyledButton = styled(Button)`
+  margin-bottom: 1rem;
+`
+
+const IN_PERSON_OPTIONS = ["in-person", "online"]
+const EVENT_TYPES = [
+  "conference",
+  "small-community-event",
+  "meetup",
+  "hackathon",
+  "other",
+]
+
+const REFERRAL_SOURCES = [
```

Comment:
Any reason to not re-use this from our inputs file? https://github.com/ethereum/esp-website/blob/master/src/utils/form-inputs.js#L242

Reply:
Ah, because it's used as keys in the lang.json files? Perhaps we should replace the existing array then.